style.css

```diff
@@ -121,3 +121,40 @@ textarea {
   position: relative;
   flex-grow: 1;
 }
+
+@media only screen and (max-width: 768px) {
+  body {
+    flex-direction: column;
+  }
+
+  textarea {
+    text-align: center;
+    padding: 10px;
+    margin: auto;
+    position: static;
+  }
+
+  .sidebar {
+    height: auto;
+    width: auto;
+  }
+
+  .sticker {
+    margin: 10px;
+    margin-bottom: 40px;
```

Comment:
You are rewriting the margin value defined right one line above.
This can lead to confuising situations. Better use shorthand here: margin: 10px 10px 40px.